browser/blueprints.py

```diff
@@ -225,17 +229,27 @@ def tool():
     }
 
     if is_trk_file(new_filename):
-        return render_template('index_track.html', filename=new_filename)
+        filetype = 'track'
+        title = 'Tracking Tool'
 
-    if is_npz_file(new_filename):
-        return render_template(
-            'index_zstack.html', filename=new_filename, settings=settings)
+    elif is_npz_file(new_filename):
+        filetype = 'zstack'
+        title = 'Z-Stack Tool'
```

Comment:
This isn't really an issue of describing the generality of the tool. As this will be seen by annotators, we should make the title something specific that reminds the user of their focus for a given job. We currently have 3 modes: one for Tracking, one for Z-Stacks, and one for Multi-Channel images. My preference would be to use these names.The only exception I see is for Z-Stacks being called "Confocal mode" given the natural synergy with biological microscopy, but that may be too esoteric for non-expert annotators.

Reply:
I'm happy to change it to whatever we agree on, but I suggest we do so in a different feature request.